.spin/bin/bootstrap

```diff
@@ -1,12 +1,19 @@
 #!/bin/bash
 
 set -ex
-source .spin/bin/env
+
+SCRIPT_PATH=$(dirname "$0")
+SCRIPT_PATH=$(cd "$SCRIPT_PATH" && pwd)
+TRUFFLERUBY_DIR=$SCRIPT_PATH/../..
```

Comment:
This seems redundant since there is the `source` below

Reply:
Ah, it's not because one needs `TRUFFLERUBY_DIR` for the `source` itself, I see.

Reply:
But I need the `SOURCE_PATH` to find the `env` file?